chart/kubeapps/templates/apprepository-deployment.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 spec:
-  replicas: 1
+  replicas: {{ .Values.apprepository.replicaCount }}
```

Comment:
I don't think controllers should have multiple replicas, can you look into this?

Reply:
Good catch, that would duplicate the sync-jobs.

Reply:
Good point, will look into it. I'd like to also know what are the consequences of having duplicated sync jobs.

Reply:
I confirmed that the controller will create duplicated jobs. I looked into other controller examples like cert-manager and etcd-operator and they seem to run by default with 1 replica, so I put it back to 1.